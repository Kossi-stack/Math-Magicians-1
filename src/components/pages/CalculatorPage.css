.calculator__page {
  padding: 5rem;
  background-color: #3AAFA9;
  min-height: 100vh;
}

.title {
  font-family: 'Inter';
  font-size: 5rem;
  font-weight: 500;
}