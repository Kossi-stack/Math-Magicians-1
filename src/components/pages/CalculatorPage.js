import React from 'react';
import './CalculatorPage.css';
import Calculator from '../Calculator';

function CalculatorPage() {
  return (
    <div className="calculator__page">
      <h1 className="title">Let&#39;s do some math 👨‍🎓👩‍🎓!</h1>
      <Calculator />
    </div>
  );
}

export default CalculatorPage;
