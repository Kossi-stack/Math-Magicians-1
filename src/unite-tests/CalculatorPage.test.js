import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
import CalculatorPage from '../components/pages/CalculatorPage';

it('CalculatorPage component is working', () => {
  const tree = renderer
    .create(
      <Router>
        <Switch>
          <Route path="/Math-Magicians" exact component={CalculatorPage} />
        </Switch>
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
