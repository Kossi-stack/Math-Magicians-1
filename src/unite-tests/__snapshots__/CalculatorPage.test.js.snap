// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalculatorPage component is working 1`] = `null`;
